lib/terraforming/resource/efs.rb

```diff
@@ -0,0 +1,35 @@
+# author: Jim Conner <snafu.x@gmail.com>
+# Nov 2016
+module Terraforming
+  module Resource
+    class EFS
+      include Terraforming::Util
+
+      def self.tf(client: Aws::EFS::Client.new)
+        self.new(client).tf
+      end
+
+      def self.tfstate(client: Aws::EFS::Client.new)
+        self.new(client).tfstate
+      end
+
+      def initialize(client)
+        @client = client
+      end
+
+      def tf
+        apply_template(@client, "tf/elastic_filesystem")
+      end
+
+      def tfstate
+        efsystems.inject({}){|resources, efs| resources}
```

Comment:
__[rubocop]__
* Space missing to the left of {.
* Space between { and | missing.
* Name `inject` block params `|a, e|`. [:ref](https://github.com/bbatsov/ruby-style-guide#reduce-blocks)
* Unused block argument - `efs`. If it's necessary, use `_` or `_efs` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Space missing inside }.

<!-- Comment by [SideCI](https://sideci.com) -->